src/components/CommunityItem/index.tsx

```diff
@@ -122,21 +137,27 @@ const CommunityItem: VFC<{ community: Amity.Community } & CommunityItemProps> =
       />
 
       <Card.Content>
-        <Paragraph style={styles.text}>{community?.description}</Paragraph>
+        <ScrollView style={styles.content}>
+          <Paragraph style={styles.text}>{community?.description}</Paragraph>
+        </ScrollView>
       </Card.Content>
-      <Card.Actions style={styles.footer}>
+
+      <Card.Actions
+        key={`Actions_${community?.membersCount}_${community?.postsCount}`}
+        style={styles.footer}
+      >
         <View style={styles.footerLeft}>
           <View style={styles.subtitleRow}>
             <MaterialCommunityIcons size={18} name="account-group" color={textColor} />
-            <Text>{community?.membersCount}</Text>
+            {!!community && <Text>{String(community?.membersCount ?? 0)}</Text>}
```

Comment:
Curious here. Why should we need to wrap this in `String` constructor? Doesn't it render the int types too?